snapcraft/sources.py

```diff
@@ -25,4 +25,3 @@
 from snapcraft.internal.sources import Tar         # noqa
 from snapcraft.internal.sources import Local       # noqa
 from snapcraft.internal.sources import Zip         # noqa
-from snapcraft.internal.sources import get         # noqa
```

Comment:
This is breaking nextcloud:
https://github.com/nextcloud/nextcloud-snap/blob/master/parts/plugins/x-php.py#L91